### DISCUSSION TITLE
Добавить метод для получения блоков главной страницы подкастов

### DISCUSSION THREAD

**Discussion by 1Finder1** [MarshalX/yandex-music-api#520] [Идеи]
Эту информацию Яндекс отдаёт легко, нужно перейти на https://api.music.yandex.net//non-music/catalogue

Будет когда-то добавлен подобный метод в библиотеке?

**Comment by MarshalX** [comment]
Когда-нибудь может и будет, но пока вы единственный человек которому это понадобилось. Могу предложить сделать pull request. Судя по структуре ответа многие модели уже готовы и присутствуют в библиотеке. В этом нет ничего сложного. Можете начать с ознакомления https://github.com/MarshalX/yandex-music-api/blob/main/CONTRIBUTING.md
